pkg/rtc/room.go

```diff
@@ -73,6 +74,16 @@ func NewRoom(room *livekit.Room, config WebRTCConfig, iceServers []*livekit.ICES
 	}
 	r.statsReporter.RoomStarted()
 	go r.audioUpdateWorker()
+
+	r.OnMetadataUpdate(func() {
+		for _, p := range r.GetParticipants() {
+			err := p.SendRoomUpdate(r.Room)
+			if err != nil {
+				logger.Warnw("failed to send room update", err, "room", room.Name, "participant", p.Identity())
+			}
+		}
+	})
```

Comment:
I wonder if I did well to put this function here (It is necessary for the test to be valid)